### PR TITLE
{vis}[foss/2019a] scikit-image v0.15.0 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by branfosj**

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f348606c26be04bf69b5f9f7532d6b41 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/825bffdd6332f92451cbf53a311e3a6c for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0305u19a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/273fe8d5ac2211905b6f3bef40f7ee53 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e2afb3bc64ef6ddbb90a03410b8ea0e5 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/4b27d8a0c7106e1a4031b1e9038f145d for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0305u19a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/feb2b975a885aed039ff00fc4b15f726 for a full test report.

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-image'
+version = '0.15.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://scikit-learn.org/stable/index.html'
```
Comment (boegel) [inline easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb]: should be `https://scikit-image.org/` (it's wrong in existing easyconfig too, we should fix that in a separate PR)

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+description = """Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
```
Comment (boegel) [inline easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb]:
```python
description = "scikit-image is a collection of algorithms for image processing."

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    }),
+    (name, version, {
+        'modulename': 'skimage',
+        'checksums': ['df111e654b47e5ea456c50553debe4c5ddd97258894c7ad3b7f2f9f10798e053'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
```
Comment (boegel) [inline easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb]: This is the default with `PythonBundle`, so `sanity_check_paths` can be removed

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb]: fixed for existing easyconfigs in #8916

Comment (branfosj) [reply easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb]: done

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb (hunk 2 above)
Comment (branfosj) [reply easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb]: done

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb (hunk 3 above)
Comment (branfosj) [reply easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb]: done

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2419.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/884f23c135ea86fe8cca611c556d232a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/56a5358779ff7fdcdad224a22429f06f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/27d995cd68bbdec0da677abbe9e25e83 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**